src/composables/transactionItemList.ts

```diff
@@ -0,0 +1,145 @@
+import { computed } from '@vue/composition-api';
+import {
+  IActiveMultisigTx,
+  IDefaultComposableOptions,
+  IDexContracts,
+  ITokenList,
+  ITransaction,
+  ITx,
+} from '../types';
+import {
+  AETERNITY_CONTRACT_ID,
+  defaultTransactionSortingCallback,
+  getInnerTransaction,
+  isTxDex,
+  pipe,
+  TX_DIRECTION,
+} from '../popup/utils';
+import { usePendingMultisigTransaction } from './pendingMultisigTransaction';
+import { useTransactionAndTokenFilter } from './transactionAndTokenFilter';
+import { i18n } from '../store/plugins/languages';
+
+interface TransactionItemList extends IDefaultComposableOptions {
+  isMultisig?: boolean,
+  token: string,
+}
+
+type ITransactionElement = ITransaction | IActiveMultisigTx
+
+export function useTransactionList({
+  store,
+  isMultisig,
+  token,
+}: TransactionItemList) {
+  const availableTokens = computed<ITokenList>(() => store.state.fungibleTokens.availableTokens);
+  const transactions = computed(() => store.state.transactions);
+  const getTxSymbol = computed(() => store.getters.getTxSymbol);
+  const getTxDirection = computed(() => store.getters.getTxDirection);
+  const getDexContracts = computed<IDexContracts>(() => store.getters.getDexContracts);
+
+  const getAccountPendingTransactions = computed<IActiveMultisigTx[]>(
```

Comment:
This is not returning `IActiveMultisigTx[]` but `ITransaction[]` as far as I know.